classes/Twig/Block/ChannelInfoBlock.php

```diff
@@ -80,12 +80,19 @@ public function render()
             $upgradeInfo['md5'] = $this->config->get('private_release_md5');
         }
 
+        /**
+         * @var string $requiredPhpVersion
+         * @var array  $psPhpCompatibilityRanges
+         */
+        extract($this->buildCompatibilityTableDisplay());
```

Comment:
I did not know this function 👍 interesting

Reply:
yes, but I removed it because of phpstan :D